### PR TITLE
Jj/update

### PR DESCRIPTION
**Pull request by moose-code**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Install dependencies of your subgraph and run the Graph CLI code generation:
+yarn && yarn codegen
+
+For auth run (get the access token from thegraph.com):
+graph auth https://api.thegraph.com/deploy/ <ACCESS_TOKEN>
+
+yarn build
+
+yarn deploy
```
Comment (moose-code) [inline README.md:9]: I am getting a subgraph validation error: [EthereumNetworkRequired]
when using yarn deploy. Please help update the readme @JasoonS

File: package.json [hunk 2]
```diff
@@ -14,51 +14,3 @@
     "@graphprotocol/graph-ts": "0.17.0"
   }
 }
-
-
-
```
Comment (moose-code) [inline package.json:19]: This was weird auto gen stuff stopping the project from actually working

File: .gitignore [hunk 3]
```diff
@@ -1,2 +1,3 @@
 node_modules
 generated
+build
```
Comment (JasoonS) [inline .gitignore:3]: You need to run `git rm -rf build` The build artifacts are all in here...

File: yarn-error.log [hunk 4]
```diff
@@ -0,0 +1,2941 @@
+Arguments:
```
Comment (JasoonS) [inline yarn-error.log:1]: Please remove this whole file. put it in the gitignore and also run `git rm ....`

**Comment by JasoonS** [comment]
image


You should rebase off master often, this branch is way behind @moose-code

**Comment by JasoonS** [comment]
image
image


No problems deploying here @moose-code

**Pull request merged**